pennylane_lightning_gpu/src/simulator/StateVectorCudaManaged.hpp

```diff
@@ -410,15 +426,21 @@ class StateVectorCudaManaged
 
     inline void applyCRX(const std::vector<std::size_t> &wires, bool adjoint,
                          Precision param) {
-        applyRX(wires, adjoint, param);
+        static const std::vector<std::string> name{{"RX"}};
+        applyParametricPauliGate(name, {wires.begin(), wires.end() - 1},
+                                 {wires.back()}, param, adjoint);
```

Comment:
Since the other calls were inline (`applyRX` and others), shouldn't they be the same as explicitly calling the function here, as the body gets replaced with the same content?

Reply:
`applyRX` and others use the inline `applyPauliRot` method now, and as the result `applyCRX/Y/Z` should directly call `applyParametricPauliGate` as `applyPauliRot` doesn't support control wires.